torchaudio/_internal/__init__.py

```diff
@@ -0,0 +1,7 @@
+from torch.hub import load_state_dict_from_url, download_url_to_file
```

Comment:
wondering why this is different from the diff. Is `_internal` directory not synced?

Reply:
looks like the linked diff corresponds to this PR, which has not been merged/synced https://github.com/pytorch/audio/pull/1959/files

Reply:
Yup, the diff and #1959 is not merged. I decided to land the minimal change here first.